### PR TITLE
Create rt-george-1712323962028359745.tweet

### PR DESCRIPTION
**Pull request by georgezgeorgez**

### CODE REVIEW

File: tweets/rt-george-1712323962028359745.tweet [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
```
Comment (0x3639) [inline tweets/rt-george-1712323962028359745.tweet]: According to research by @Sol-Sanctum we need to remove the `---` if you don't plan to add RT comments.  

retweet: <link>

**Comment by sol-znn** [comment]
Good suggestion, invalid format

https://github.com/zenon-network/twitter/pull/24#discussion_r1355388041

**Comment by georgezgeorgez** [comment]
Can we modify
https://github.com/zenon-network/twitter#retweeting-and-quote-retweeting

to have and example of plain RT?

**Comment by georgezgeorgez** [comment]
https://github.com/zenon-network/twitter/pull/27

**Comment by georgezgeorgez** [comment]
Updated

**Review by 0x3639 — APPROVED** [review]

**Comment by sumamu** [comment]
Very good tweet. I'd approve.

**Review by mehowz — APPROVED** [review]

**Comment by 0x3639** [comment]
I have no idea how this worked.  This should have failed.

**Comment by coinselor** [comment]
Quote retweets work. That's basically all we did for the Tribute tweet. 

We need to add something more meaningful than the word "retweet:" if we are doing a quote retweet (the only type of retweet we seem to be able to pull off given the API restrictions). Think of adding at least a one liner for context, or providing commentary. 

In this case, we could have mentioned the Zenon Token Standard and link to another tweet that explains/mentions it.

I'm assuming we can't even edit the tweet through the API since we don't even have Twitter Blue.

**Comment by sol-znn** [comment]
> I'm assuming we can't even edit the tweet through the API since we don't even have Twitter Blue.

Can't edit, but 0x3639 has the ability to delete.

**Pull request merged**